src/components/Avatar/AvatarGroup.tsx

```diff
@@ -111,21 +109,25 @@ export const AvatarGroup: React.FC<AvatarGroupProps> = React.forwardRef(
       </ConditionalWrapper>
     );
 
-    if (avatarListProps && maxCount && maxCount < numChildren) {
+    if (avatarListProps) {
+      const inlineCount = maxCount
```

Comment:
Please apply the same fixes to the `if (!avatarListProps)` condition too. That way the maxCount won't be required to render if the group only contains a single item.

Reply:
have made the recommended changes dylan.